akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/ReplicatedEventSourcingSpec.scala

```diff
@@ -461,5 +495,38 @@ class ReplicatedEventSourcingSpec
       r2 ! StoreMe("from r2", probe.ref)
       eventProbeR1.expectMessageType[EventAndContext].event shouldEqual "from r2"
     }
+
+    "migrate from EventSourcedBehavior to ReplicatedEventSourcedBehavior" in {
+      val entityId = nextEntityId
+      val probe = createTestProbe[Done]()
+      // first ordinary EventSourced
+      val es1 = spawn(nonReplictedTestBehavior(entityId))
+      es1 ! StoreMe("from es1", probe.ref)
+      probe.expectMessage(Done)
+      es1 ! Stop
+
+      // then migrate to ReplicatedEventSourced, one replica must keep the same persistenceId
+      // as the original EventSourced, i.e. without replicaId
```

Comment:
The replicaId is part of the persistenceId so I think it's easiest to let the original remain as is. I used `""` as indication of such replicaId.

Reply:
Might be worth having a `EmptyReplicaId` or something for it